ntbs-service/Jobs/HangfireJobScheduler.cs

```diff
@@ -11,7 +11,13 @@ public static void ScheduleRecurringJobs()
                 "user-sync",
                 job => job.Run(JobCancellationToken.Null),
                 Cron.Daily(3),
-                TimeZoneInfo.Local);            
+                TimeZoneInfo.Local);
+            
+            RecurringJob.AddOrUpdate<CloseInactiveNotificationsJob>(
```

Comment:
Sorry - this is going to need to be added to the config which has just been merged into master.